rollup.config.js

```diff
@@ -48,20 +55,6 @@ const getPlugins = (env) => {
         }
       }
     }),
-    // not add @babel/runtime import for development
-    (process.env.NODE_ENV !== 'development' && babel({
-      babelHelpers: 'runtime',
-      presets: [
-        '@babel/preset-env'
-      ],
-      plugins: [
-        // https://babeljs.io/docs/en/babel-plugin-transform-runtime#corejs
-        ['@babel/plugin-transform-runtime', { 
```

Comment:
Why remove this line? I think some functions, like `Object.entries` will need this runtime plugin to compatible with old browsers.